Makefile

```diff
@@ -154,6 +157,9 @@ build/$(CONVERTER_JAR): $(CONVERTER_SOURCES) src/converter/MANIFEST.MF
 %.class: %.java
 	$(JAVAC) -g:none -source $(JAVAC_RELEASE_VERSION) -target $(JAVAC_RELEASE_VERSION) $(*D)/*.java
 
+%.html.h: %.html
+	hexdump -v -e '1/1 "%u,"' $^ > $@
```

Comment:
Not sure if it the best option

Reply:
Have you tried `.incbin` directive in C++ inline asm?
It would be nice to avoid intermediate files.

Reply:
Indeed, it is a very fun way to include data)